devstack/plugin.sh

```diff
@@ -0,0 +1,148 @@
+
+# Devstack plugin code for Calico
+# ===============================
+
+function install_configure_etcd {
+
+    # Install etcd from package sources (=> our PPA).
+    install_package etcd
+
+    # Stop the etcd service:
+    sudo service etcd stop || true
+
+    # Delete any existing etcd database:
+    sudo rm -rf /var/lib/etcd/*
```

Comment:
This is scary.  If I was running etcd for some other purpose already, this would nuke it.  Would it be better to run the devstack etcd cluster from a dedicated directory?